AsyncDisplayKitTests/ASTableViewTests.m

```diff
@@ -72,6 +72,11 @@ - (ASCellNode *)tableView:(ASTableView *)tableView nodeForRowAtIndexPath:(NSInde
   return nil;
 }
 
+- (ASDataControllerCellNodeBlock)tableView:(ASTableView *)tableView nodeBlockForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+  return nil;
+}
```

Comment:
Simple, but beautiful.  Thanks for starting test-first, and even these simple ones achieve two important goals - 1) testing common cases, of actual relevance ; and 2) provide an environment / hint to others that we can expand testing here, and there's a spot just for you if you're adding node block tests :)